### PR TITLE
add ogr source doc

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Filter in der OgrSource](https://trello.com/c/MmWS8ix3/607-filter-in-der-ogrsource)

File: src/operators/expression.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ When the temporal resolution is months, our output NDVI will also be a monthly t
...
 | `outputNoDataValue` | `Number`                                         | NO DATA value for the output                                                                                | <pre><code>-2</code></pre>                                                                               |
 | `outputMeasurement` | [`Measurement`](/datatypes/measurement.md)       | Description about the output                                                                                | <pre><code>{<br>&nbsp;&nbsp;"type": "continuous",<br>&nbsp;&nbsp;"measurement": "NDVI"<br>}</code></pre> |
-| `mapNoData`         | `Boolean`                                        | Should NO DATA values be mapped with the `expression`? Otherwise, they are mapped automatically to NO DATA. | <pre><code>false</code></pre>                                                                            |
+| `mapNoData`         | `bool`                                        | Should NO DATA values be mapped with the `expression`? Otherwise, they are mapped automatically to NO DATA. | <pre><code>false</code></pre>                                                                            |
```
Comment (ChristianBeilschmidt) [inline src/operators/expression.md:21]: don't we use Boolean throught this doc?

File: test.http [hunk 2]
```diff
@@ -0,0 +1,214 @@
+POST http://localhost:3030/user
```
Comment (ChristianBeilschmidt) [inline test.http:1]: I guess you should remove this file.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: test.http (hunk 2 above)
Comment (michaelmattig) [reply test.http]: b0a3ee81a27ec78310ed5b9683fd9aafe1f451d8

File: src/operators/expression.md (hunk 1 above)
Comment (michaelmattig) [reply src/operators/expression.md:21]: there were more occasions of bool than Boolean throughout the pages so I used that one

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**